src/rdkafka_assignor.c

```diff
@@ -364,14 +369,17 @@ rd_kafka_assignor_run (rd_kafka_cgrp_t *rkcg,
                                    rkas->rkas_opaque);
 
         if (err) {
-                rd_kafka_dbg(rkcg->rkcg_rk, CGRP, "ASSIGN",
+                rd_kafka_dbg(rkcg->rkcg_rk, CGRP|RD_KAFKA_DBG_ASSIGNOR,
```

Comment:
not suggesting changes, but musing it would better if there was the concept of debug context groupings. i.e. you just specify ASSIGNOR here and that would be a child of CGRP. As a user, this gives me more certainty/clarity on what i'm getting debug messages for.

Reply:
The idea here is to be able to debug only the assignors and not the cgrp stuff, but this debug should be shown for either.